lib/shopify_api/resources/collection.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module ShopifyAPI
+  class Collection < Base
+    include Events
```

Comment:
do we need to include `Events`?

Reply:
I think yes, we do need to keep support for `Events`. We will also have to create PR in `Shopify/shopify` to add support for `Collection.events`.